charts/fluent-bit/values.yaml

```diff
@@ -351,8 +352,8 @@ config:
         Daemon Off
         Flush {{ .Values.flush }}
         Log_Level {{ .Values.logLevel }}
-        Parsers_File parsers.conf
-        Parsers_File custom_parsers.conf
+        Parsers_File /fluent-bit/etc/parsers.conf
```

Comment:
Yeah, good change - I've had issues relying on relative paths in the past so better to be explicit. People can change the working directory and it'll bork.

Reply:
It's actually needed now due to the current Fluent Bit OCI image config file layout issue.

Reply:
Ah yes, we have moved things into `/conf`. 

Need to make sure that is clear somewhere in the new release - some people may be custom mounting as well I guess or looking for things during debug.

Reply:
We don't really have any way of doing this currently, that's why we need a better release process.